### DISCUSSION TITLE
Issues with Modal over size or scrolling

### DISCUSSION THREAD

**Discussion by ScarecrowFu** [iddan/react-spreadsheet#190] [Q&A]
is there any solutions for over size in modal?

![http://90fyl.com/media/%E5%9B%BE%E7%89%87/alan/None/QQ20210312-160145.png](http://90fyl.com/media/%E5%9B%BE%E7%89%87/alan/None/QQ20210312-160145.png)

[http://90fyl.com/media/%E5%9B%BE%E7%89%87/alan/None/QQ20210312-160145.png](http://90fyl.com/media/%E5%9B%BE%E7%89%87/alan/None/QQ20210312-160145.png)

**Comment by iddan** [comment]
Can you provide a codesandbox with your issue?

**Comment by NoobTW** [comment]
FYI, https://brdys.csb.app/

And here is the source: https://codesandbox.io/s/brdys

**Comment by NoobTW** [comment]
A quick workaround might be wrapping with a `overflow: scroll` div.

```jsx
<div style={{ maxWidth: 100%, overflow: 'scroll' }}>
    <Spreadsheet data={dataWithManyColumns} />
</div>
```
